Packages/doc/developers.rst

```diff
@@ -412,3 +412,16 @@ Windows 10 S30 MIES CI:
 - ITC18-USB hardware, 2 AD/DA channels are looped
 - MCC demo amplifier only
 - Latest required nightly version of Igor Pro 8
+
+Branch naming scheme
+~~~~~~~~~~~~~~~~~~~~
+
+For making code review easier we try to follow a naming scheme for branches behind PRs.
+
+Scheme: ``$prefix/$pr-$text(-backport)?``
+
+Where ``$prefix`` is one of ``feature``/``bugfix``, ``$pr`` is the number of the soon-to-be-created pull request and
+``$text`` a user defined descriptive text. ``-backport`` must be present for PRs against release branches only.
+
+Contributers are encouraged to install the ``pre-push`` git hook from the tools directory. The script
+``tools/nextFreePRNumber.sh`` can get the soon-to-be-created PR number on the commandline (requires curl and jq) as well.
```

Comment:
@t-b this seems to make it a lot harder to for folks who know Igor well, but not other languages, to make PRs. Do you agree?

Reply:
@timjarsky Well the `encouraged` should mean that this is a `can` rule. So contributors can also make PRs which don't follow the naming scheme. They also don't have to program anything in other programming languaugse as this is already done. Does that make sense?

This mostly just says that me and Michael fill follow the naming scheme ;)